modules/animation/src/animationcontroller.cpp

```diff
@@ -129,6 +129,9 @@ AnimationController::AnimationController(Animation& animation, InviwoApplication
     playOptions.setCollapsed(true);
     addProperty(playOptions);
 
+    framesPerSecond.onChange([this]() { setPlaySpeed(framesPerSecond); });
```

Comment:
Should not setPlaySpeed update frames per second too? Don't know why there is two members with same state...

Reply:
`deltaTime_` is the inverse of framesPerSecond and also holds the playback direction (signed delta).

Reply:
going for OrdinalRefProperty to hold the state only once.